content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,26 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/aws-load-balancer-security-groups (deprecated) {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f, sg-8725gr62r"`
+
+Used on: Service
+
+The cloud controller manager integration with AWS elastic load balancing configures
```

Comment:
Are you sure this annotation is added by the CCM?

Reply:
This annotation as per my knowledge is configured by AWS Load Balancer controller. I am not sure, Would like to know your suggestion. How about i rewrite it like this 

```
The official Kubernetes integration with AWS elastic load balancing configures
this annotation to specify a comma seperated list of frontend securityGroups you want
to attach to an NLB. Both name and ID of securityGroups are supported where name matches
a `Name` tag, not the `groupName` attribute.

When the annotation is not present, the AWS Load Balancer controller will automatically create one security group.
When this annotation is present, all other previously assigned security groups are replaced by this list.
Also you need to configure the security groups on your target instances/ENIs to allow
inbound traffic from the load balancer.
```

Reply:
I have added the AWS Load Balancer Controller in the later lines to specify which controller we are talking about.